ytsaurus_helpers.go

```diff
@@ -12,12 +12,16 @@ import (
 
 // Lower level functions for reusing in tests.
 
-func doGetAllYtsaurusUsers(ctx context.Context, client yt.Client) ([]YtsaurusUser, error) {
+const (
+	bannedSinceAttributeName = "banned_since"
+	bannedAttributeName      = "banned"
+	sourceTypeAttributeName  = "source_type"
```

Comment:
We don't need to store that in ytsaurus, we have it in config.

Reply:
No, we don't. In config we have only `SourceAttributeName` (for "azure" or "source" values).

Reply:
my bad, but still think since source is configured in the app we can delegate handling source-fields to that source without storing it in attributes.